toolbox/benchmarking.py

```diff
@@ -16,7 +16,7 @@ def download_coco_dataset(node_hostname, namespace="default", pvc_name=None, mir
             namespace: Name of the namespace in which the resources will be created.
             pvc_name: Name of the PVC that will be create to store the dataset files.
             mirror_base_url: Optional base URL where to fetch the dataset
-            client_cert: Optional tath to the client cert to use for accessing the base URL.
+            client_cert: Optional path to the client cert to use for accessing the base URL.
```

Comment:
This could have been it's own commit

Reply:
indeed, but it's the entrypoint of the role, so same topic 🙃